.github/workflows/post_merge_integration.yml

```diff
@@ -207,6 +207,10 @@ jobs:
         run: npm config set @psu:registry https://artifacts.developer.gov.bc.ca/artifactory/api/npm/pe1e-psu-npm-local/ && npm config set //artifacts.developer.gov.bc.ca/artifactory/api/npm/pe1e-psu-npm-local/:_authToken ${{ secrets.ARTIFACTORY_SVCACCT_TOKEN }}
       - name: Install node dependencies (web)
         working-directory: ./web
+      - name: enable corepack
+        run: corepack enable
+      - name: use new yarn
+        run: yarn set version berry
```

Comment:
Do we need to setup yarn for scoped artifactory packages here like was done in `integration.yml`?

Reply:
ah good catch

Reply:
fixed